docs/book/src/quick-start.md

```diff
@@ -57,6 +57,29 @@ cd $GOPATH/src/example
 kubebuilder init --domain my.domain
 ```
 
+If there is an error similar to `dial tcp x.x.x.x:443: i/o timeout`, you need to set go proxy.
+Go version >= 1.13
+
+```bash
+go env -w GO111MODULE=on
+go env -w GOPROXY="https://goproxy.io,direct"
```

Comment:
Go 1.13 already set these values by default. However, if you might would like to help others I think it the place would be tracking and issue with the solution and/or creating a FAQ section in the docs instead of it. WDYT?